### PR TITLE
doc: update README, make it easy to read and follow

### PR DESCRIPTION
**Pull request by DoodleBears**
- Rearrange chapter
- Add tutorial on deploying API server
- Add Emoji

# Screenshots
image

---

image

### CODE REVIEW

**Review by imoneoi — APPROVED** [review]
LGTM! Thanks for your contribution!

**Comment by DoodleBears** [comment]
> LGTM! Thanks for your contribution! You can merge it when ready!

Thanks, I have nothing to update, let's merge it.

**Comment by imoneoi** [comment]
@DoodleBears btw can you make a bit modifications as above?

**Comment by DoodleBears** [comment]
> @DoodleBears btw can you make a bit modifications as above?

What modifications

**Comment by imoneoi** [comment]
> > @DoodleBears btw can you make a bit modifications as above?
> 
> What modifications, since the content wrote by author have their intention, I did not revise the content that much

@DoodleBears See code review

**Review by DoodleBears — COMMENTED** [review]
Sorry, I did not see the comment in code review?

**Comment by imoneoi** [comment]
> Sorry, I did not see the comment in code review?

Can you see it here? https://github.com/imoneoi/openchat/pull/70/files/0bccc1438e99a26afd8059aba121bf8babde8d25

File: README.md
Comment (DoodleBears) [inline README.md:1]: Can not see

**Comment by imoneoi** [comment]
@DoodleBears Merged, great thanks! I've made a little modifications.

BTW I tested this command should work. Will it work in your env?

```
python -m ochat.serving.openai_api_server --model openchat/openchat_3.5
```

**Comment by DoodleBears** [comment]
Yes. it works on my env (WSL)

**Pull request merged**